compose/gpu-support.md

```diff
@@ -0,0 +1,170 @@
+---
+description: GPU support in Compose
+keywords: documentation, docs, docker, compose, GPU access, NVIDIA, samples
+title: Enabling GPU access with Compose
+---
+
+Compose services can define GPU device reservations if the Docker host contains such devices and the Docker Daemon is set accordingly. For this, make sure to install the [prerequisites](../_data/engine-cli/docker_run.yaml) if you have not already done so.
+
+The examples in the following sections focus specifically on providing service containers access to GPU devices with Docker Compose (either `docker-compose` or `docker compose` commands can be used).
+
+### Use of service `runtime` property from Compose v2.3 format (legacy)
+
+Docker Compose v1.27.0+ switched to using the Compose Specification schema which is a combination of all properties from 2.x and 3.x versions. This re-enabled the use of service properties as [runtime](https://docs.docker.com/compose/compose-file/compose-file-v2/#runtime) to provide GPU access to service containers. However, this does not allow to have control over specific properties of the GPU devices.
+
+```yaml
+services:
+  test:
+    image: nvidia/cuda:10.2-base
+    command: nvidia-smi
+    runtime: nvidia
+
+```
+
+### Enabling GPU access to service containers
+
+Docker Compose v1.28.0+ allows to define GPU reservations using the [device](https://github.com/compose-spec/compose-spec/blob/master/deploy.md#devices) structure defined in the Compose Specification. This provides more granular control over a GPU reservation as custom values can be set for the following device properties: 
+
+- [capabilities](https://github.com/compose-spec/compose-spec/blob/master/deploy.md#capabilities) - value specifies as a list of strings (eg. `capabilities: [gpu]`) This field must be set in the Compose file, otherwise an error will be returned on service deployment.
```

Comment:
```suggestion
- [capabilities](https://github.com/compose-spec/compose-spec/blob/master/deploy.md#capabilities) - value specifies as a list of strings (eg. `capabilities: [gpu]`) You must set this field in the Compose file. Otherwise, it returns an error on service deployment.
```